Backend/src/utils/filesystem.py

```diff
@@ -36,3 +40,28 @@ def read_meta_info(file_name: str) -> dict:
             raise ValueError(f"Error parsing JSON from file '{meta_info_path}': {e}")
 
     return meta_info_data
+
+def download_qaqc_source_dataset() -> pd.DataFrame:
+    """
+    Downloads the qaqc data, returns the .csv file.
+    """
+    dataset_url = "https://svn.spraakbanken.gu.se/sb-arkiv/pub/trec/swe_qaqc_train.csv"
+    response = requests.get(dataset_url)
+    if response.status_code == 200:
+        csv_content = StringIO(response.content.decode('utf-8'))
+        df = pd.read_csv(csv_content)
+        return df
+    else:
+        raise Exception(f"Failed to download QAQC dataset, status code: {response.status_code}")
+    
+
+def get_qaqc_info_dict() -> dict:
```

Comment:
The function `get_qaqa_info_dict` could also be added to the same *new* file as `download_qaqc_source_dataset`.

Reply:
done